client/src/components/WeekListItem/WeekListItem.js

```diff
@@ -22,7 +22,7 @@ const Diamond = styled.div`
   padding-bottom: 5px;
 `;
 
-const TextWeek = styled.p`
+const TextWeek = styled.a`
   color: var(--text-color);
   padding-bottom: 1px;
   /* 1px looked more aligned with the hr line. */
```

Comment:
Do you neeed this comment?

Reply:
somebody else requested comments for weird px numbers, that's why :D